src/stylesheets/elements/typography/_links.scss

```diff
@@ -0,0 +1,68 @@
+.usa-link {
```

Comment:
This should be combined with existing _links.scss in src/stylesheets/elements/_links.scss